remove glusterfs references from the docs

humblec opened the pull request:
we have removed glusterfs in-tree driver in this release (v1.26) via https://github.com/kubernetes/kubernetes/pull/112749 .
This PR remove the references from the docs.

humblec commented:
Cc @tengqm @sftim @xing-yang

tengqm commented:
/lgtm

sftim reviewed (commented):
Thanks @humblec 

I'd like to include `glusterFS` somewhere in the volumes page, so that if people search the docs, they find a match. A single sentence explaining that Kubernetes v1.25 was the last minor release with in-tree support is enough.

Please don't update any of the pages with `auto-generated: true` in their front matter. We'll cover those separately.

humblec commented:
> Thanks @humblec
> 
> I'd like to include `glusterFS` somewhere in the volumes page, so that if people search the docs, they find a match. A single sentence explaining that Kubernetes v1.25 was the last minor release with in-tree support is enough.

@sftim indeed thats a good thought. I have added a note in volumes guide about the same. 
> 
> Please don't update any of the pages with `auto-generated: true` in their front matter. We'll cover those separately.

sure

sftim reviewed (commented):
One more thing. Sorry.

humblec commented:
> One more thing. Sorry.

nw @sftim :+1: , I committed those changes. ptal. thanks.

leonardpahlke commented:
Looks like this is almost ready, @humblec this needs a rebase & another checkin by @sftim if you find the time :)

humblec commented:
sure @leonardpahlke . let me do one more rebase :) ..

leonardpahlke commented:
K8s v1.26 is scheduled for less than a week, so we need to get this done as soon as possible! 🙏 
cc @krol3

humblec commented:
> K8s v1.26 is scheduled for less than a week, so we need to get this done as soon as possible! pray cc @krol3

@leonardpahlke @krol3 the PR has been rebased. we should be good to go . thanks !

tengqm commented:
@humblec Looks like the gate is not happy with the new commit? Could you pls try it again?

humblec commented:
> @humblec Looks like the gate is not happy with the new commit? Could you pls try it again?

@tengqm I just rebased the PR and fixed the conflicts. do we have pointer on what exactly causing the failure?

tengqm commented:
@humblec It smells like a transient netlify build timeout issue.
You may want to recommit this PR to trigger a new gate build.

humblec commented:
> @humblec It smells like a transient netlify build timeout issue. You may want to recommit this PR to trigger a new gate build.

Cool. I have retriggered the tests @tengqm .. Thanks :+1:

xing-yang commented:
/lgtm

tengqm commented:
/label tide/merge-method-squash

tengqm commented:
/approve
